test/sdk/MathTest.ooc

```diff
@@ -0,0 +1,30 @@
+use ooc-unit
+import math
+
+MathTest: class extends Fixture {
+	init: func {		
+		super("Math")
+		tolerance := 1.0e-5f		
+		this add("Modulo", func {
+			intValuePos := 22
```

Comment:
I would love to also have tests where the dividend is closer to zero then the divider.